app/controllers/v1/supplemental_claims_controller.rb

```diff
@@ -85,10 +87,12 @@ def handle_personal_info_error(e)
       raise
     end
 
-    def process_submission
+    def process_submission # rubocop:disable Metrics/MethodLength
```

Comment:
It would be great to be able to remove this rubocop disable. Can you pull out some logic that doesn't _need_ to be in this method? You could make some private methods to parse the response body and extract data from the request, etc.

Reply:
Thanks, I've refactored the method accordingly and removed the ruboco disable. 👍